chronicle/Views/EnrolledView.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  EnrolledView.swift
+//  EnrolledView
+//
+//  Created by Giulia Campana on 8/12/21.
+//
+
+import SwiftUI
+
+struct EnrolledView: View {
+    var enrollmentViewModel: EnrollmentViewModel
+    let defaults = UserDefaults.standard
+    
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading) {
+                AppHeader()
+                Text("Study ID:").padding(.bottom)
+                Text(defaults.string(forKey: UserSettingsKeys.studyId) ?? "").padding(.bottom)
+                Text("Participant ID:").padding(.bottom)
+                Text(defaults.string(forKey: UserSettingsKeys.participantId) ?? "").padding(.bottom)
+                if (defaults.string(forKey: UserSettingsKeys.organizationId) != nil) {
+                    Text("Organization ID:").padding(.bottom)
+                    Text(defaults.string(forKey: UserSettingsKeys.organizationId) ?? "").padding(.bottom)
+                }
```

Comment:
I remember approving the other pull request where the org id is made optional. I think we should revert that and make the org id required.

Reply:
okay, gotcha—so just to clarify, we are making the org id required on the enrollment form, as well as displaying it by default here on the enrolled screen?